dist/quick-release-notes/quick-release-notes.go

```diff
@@ -0,0 +1,431 @@
+		if releaseMetaData.ReleaseVersionTagMap[releaseVersion] == releaseMetaData.TagReleaseVersionMap[tagVersion] {
+			repoOwner, repoName := splitAndEncodeURLPath(repoPath)
+			requestURL := fmt.Sprintf("https://api.github.com/repos/%v/%v/releases", repoOwner, repoName)
+			postRequest := releaseRequest{
+				authToken:     authToken,
+				repoPath:      repoPath,
+				requestType:   "POST",
+				tagForRelease: tagVersion,
+				versionTitle:  releaseMetaData.TagReleaseVersionMap[releaseVersion],
+				releaseNote:   releaseMetaData.ReleaseVersionNoteMap[releaseVersion],
+			}
+			createReleaseWaitGroup.Add(1)
+			go updateOrCreateGitHubRelease(postRequest, requestURL, createReleaseChannel, createReleaseWaitGroup)
+			spawnedThread = true
+		}
+	}
+	if spawnedThread {
+		handleChannels(createReleaseChannel, createReleaseWaitGroup)
+	} else {
+		close(createReleaseChannel)
+	}
+}
+
+func updateReleasesIfChanged(releaseMetaData releaseMetaData, authToken string, repoPath string) {
+	updateReleaseWaitGroup := &sync.WaitGroup{}
+	updateChannel := make(chan bool, len(releaseMetaData.ReleaseVersionTagMap))
+
+	repoOwner, repoName := splitAndEncodeURLPath(repoPath)
+	releases := getReleases(authToken, repoPath)
+
+	spawnedThread := false
+	for _, release := range releases {
+		if release.Body != releaseMetaData.ReleaseVersionNoteMap[release.Name] {
+			requestURL := fmt.Sprintf("https://api.github.com/repos/%v/%v/releases/%v", repoOwner, repoName, release.ID)
+			patchRequest := releaseRequest{
+				authToken:     authToken,
+				repoPath:      repoPath,
+				requestType:   "PATCH",
+				tagForRelease: release.TagName,
+				versionTitle:  release.Name,
+				releaseNote:   releaseMetaData.ReleaseVersionNoteMap[release.Name],
+			}
+			updateReleaseWaitGroup.Add(1)
+			go updateOrCreateGitHubRelease(patchRequest, requestURL, updateChannel, updateReleaseWaitGroup)
+			spawnedThread = true
+		}
+	}
+	if spawnedThread {
+		handleChannels(updateChannel, updateReleaseWaitGroup)
+	} else {
+		close(updateChannel)
+	}
+}
+
+func validateTagsHaveReleases(releaseTagValidationInput releaseTagValidationInput) releaseMetaData {
+	releaseMetaData := releaseMetaData{
+		ReleaseVersionNoteMap: make(map[string]string),
+		ReleaseVersionTagMap:  make(map[string]string),
+		TagReleaseVersionMap:  make(map[string]string),
+	}
+	for i, releaseVersion := range releaseTagValidationInput.changeLog.versions[:] {
+		tagVersionForMap := ""
+		releaseVersionHasTag := false
+		for _, tagVersion := range releaseTagValidationInput.tags[:] {
+			if releaseVersion.number == tagVersion.Name {
+				releaseVersionHasTag = true
+				tagVersionForMap = tagVersion.Name
+			}
+		}
+		if releaseVersionHasTag == false {
+			fmt.Println(redColor+"WARNING: release", releaseVersion, "missing tag"+resetColor)
+		}
+		if releaseVersionHasTag {
+			releaseMetaData.ReleaseVersionTagMap[releaseVersion.number] = tagVersionForMap
```

Comment:
Because `tagVersion.Name` is equal to `releaseVersion.number` (line 223), we could write `releaseVersion.number` on this line and get rid of the `tagVersionForMap` variable.